### PR TITLE
INLG 2023 ingestion

### PR DESCRIPTION
**Pull request by anthology-assist**

### CODE REVIEW

File: data/xml/2023.dstc.xml [hunk 1]
```diff
@@ -18,6 +18,7 @@
       <month>September</month>
       <year>2023</year>
       <venue>dstc</venue>
+      <venue>ws</venue>
```
Comment (anthology-assist) [inline data/xml/2023.dstc.xml:21]: @mjpost So we have two tags -- `dstc` (workshop) venue tag and `ws` tag?

Comment (mjpost) [reply data/xml/2023.dstc.xml:21]: Yes. Every volume has a tag for its venue. Then, if it is a workshop, we also add ws. No volume can have *just* ws (except old ones that we haven't fixed yet). 

This was tag replaces back linking from the YYYY.ws.xml file.

Comment (mjpost) [reply data/xml/2023.dstc.xml:21]: Also this needs to be added to each workshop in the PR.

File: data/xml/2023.cs4oa.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      <booktitle>Proceedings of the 1st Workshop on CounterSpeech for Online Abuse (CS4OA)</booktitle>
+      <editor><first>Yi-Ling</first><last>Chung</last></editor>
+      <editor><first>Helena</first><last>Bonaldi</last></editor>
+      <editor><first>Gavin</first><last>Abercrombie</last></editor>
+      <editor><first>Marco</first><last>Guerini</last></editor>
+      <publisher>Association for Computational Linguistics</publisher>
+      <address>Prague, Czechia</address>
+      <month>September</month>
+      <year>2023</year>
+      <venue>cs4oa</venue>
```
Comment (mjpost) [inline data/xml/2023.cs4oa.xml:14]: This one still needs `<venue>ws</venue>`

**Comment by anthology-assist** [comment]
@mjpost Could we get this approved?

**Review by mjpost — APPROVED** [review]

**Pull request merged**